components/datetime/src/neo_marker.rs

```diff
@@ -75,6 +75,69 @@
 //! );
 //! ```
 //!
+//! ## Hour Cycle
+//!
+//! Hours can be switched between 12-hour and 24-hour time via the `u-hc` locale keyword.
+//!
+//! ```
+//! use icu::calendar::Gregorian;
+//! use icu::calendar::Time;
+//! use icu::datetime::neo::NeoOptions;
+//! use icu::datetime::neo::TypedNeoFormatter;
+//! use icu::datetime::neo_marker::NeoHourMinuteMarker;
+//! use icu::datetime::neo_skeleton::NeoSkeletonLength;
+//! use icu::datetime::NeverCalendar;
+//! use icu::locale::locale;
+//! use writeable::assert_try_writeable_eq;
+//!
+//! // By default, en-US uses 12-hour time and fr-FR uses 24-hour time,
+//! // but we can set overrides.
+//!
+//! let formatter =
+//!     TypedNeoFormatter::<NeverCalendar, NeoHourMinuteMarker>::try_new(
+//!         &locale!("en-US-u-hc-h12").into(),
```

Comment:
don't use default overrides?
```suggestion
//!         &locale!("en-US").into(),
```

Reply:
I want to test that `en-US` and `en-US-u-hc-h12` both work. There are other pre-existing tests for `en-US`.